style/index.css

```diff
@@ -18,7 +18,24 @@
   font-size: var(--jp-content-font-size1);
 }
 
-body[data-jp-theme-light='false'] .h5web-root {
+/* Deactivate h5web dark mode handling */
+body .h5web-root > div {
+  filter: none;
+}
+
+body .h5web-root > div [data-keep-colors],
+body .h5web-root > div [data-keep-canvas-colors] canvas {
+  filter: none;
+}
```

Comment:
Good thinking using `filter: none` 💯 

I think you could simplify all this with:

```css
.h5web-root > div,
.h5web-root canvas {
  filter: none !important;
}
```

Reply:
I gave a try but this or variations do not work: either it does not disable the `invert` or it prevents the application of the `invert` when switching to JLab dark mode.

I still simplified the selectors by merging the rule and removing `body`